quick_tour/interacting_with_the_index.ipynb

```diff
@@ -121,7 +121,7 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    fetch_results = index.fetch(ids=["A", "B", "C"])
</pre><p>Let's remove C from the fetch</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pinecone-io/examples/pull/42/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>